english/cowboy/cowboy.md

```diff
@@ -1,5 +1,8 @@
 ... Once you feel your best bib and tucker are all correct, you strut into the
 bar, push the nearest deadbeat and start the meanest brawl the ruff'n'tumble
 tavern has ever seen.
-You grab your aggressor and ask him "Where are we ?", he 
-replies ["Texas"](../texas/texas.md)
\ No newline at end of file
+You grab your aggressor and ask him "Where are we ?", he
+replies ["Texas"](../texas/texas.md)
+
+replies, with a distant look in his eye, [another dimension](english/dimensions/dimension.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.